torch/_inductor/lowering.py

```diff
@@ -297,6 +297,26 @@ def wrapped(*args, **kwargs):
         return out
 
     aten_fn = get_overloads(aten_fn)
+    core_aten_ops = core_aten_decompositions()
+
+    for fn in aten_fn:
+        if not isinstance(fn, torch._ops.OpOverload):
+            continue
+
+        py_kernels = None
+        if hasattr(fn, "default"):
+            fn = fn.default
+        if hasattr(fn, "py_kernels"):
+            py_kernels = fn.py_kernels
+
+        if py_kernels is not None and torch._C.DispatchKey.Autograd in py_kernels:
+            has_decomp = len(get_decompositions([fn])) > 0
```

Comment:
It must have a decomp if it has an Autograd in py_kernels, right?

Reply:
my point is, you should be able to assert here
```suggestion
           assert len(get_decompositions([fn])) > 0
```
and throw an error if the function is register with the Python dispatcher but is not registered as a decomp